model.py

```diff
@@ -6580,7 +6580,7 @@ def lookup_by_token(self, _db, data_source, type, token,
             return None
 
     @classmethod
-    def lookup_by_temporary_token(cls, _db, data_source, type, token):
+    def lookup_and_expire_temporary_token(cls, _db, data_source, type, token):
```

Comment:
Made this change as per Amy's suggestion on an earlier branch. I don't know if this method is used at all outside of its test.